maintainers/maintainer-list.nix

```diff
@@ -15324,6 +15324,12 @@
     githubId = 1368952;
     name = "Pedro Lara Campos";
   };
+  peefy = {
```

Comment:
Please add yourself to the maintainer list in a separate commit, as described in the [manual](https://nixos.org/manual/nixpkgs/stable/#var-meta-maintainers).

You can `git reset origin/master`, then create 2 separate commits, and force-push to rewrite the current commit history of this branch with `git push --force-with-lease` to do so.

Reply:
Sorry, confused. @RossComputerGuy asked me to merge two commits in this PR. Ref: https://github.com/NixOS/nixpkgs/pull/302497

Reply:
Having one PR is correct, but separate the changes into 2 commits. I think that's what he was trying to say too.

Reply:
fixed. PTAL

Reply:
Thanks! Please reword it to `maintainers: add peefy` though, as per the [convention](https://nixos.org/manual/nixpkgs/stable/#var-meta-maintainers)

Reply:
Thanks. Done.